dist/ci/docker-compose-test.sh

```diff
@@ -13,13 +16,8 @@ until curl http://api:3000/about 2>/dev/null ; do
 done
 
 cd /code
-for file in tests/*_tests.py; do
-  if ! test -f /code/.without-coverage; then
-    COVER_ARGS="--with-coverage --cover-xml --cover-package=. --cover-inclusive"
-  fi
-  echo "running tests from $file..."
-  run_as_tester nosetests $COVER_ARGS -c .noserc -s $file
-done
+# TODO: Review bot test failed without log-level set to debug
+run_as_tester pytest --log-level=DEBUG --cov=. tests/*_tests.py
```

Comment:
this log level is strange...maybe nosetests has it by default and it can be reason why pytest failed in that review bot test.